### PR TITLE
Proofread `changelogs/unreleased`

### PR DESCRIPTION
**Pull request by arctic-dreamer**
All changes are made according to the rules described in "[Contributing > How to write release notes](https://www.tarantool.io/en/doc/latest/contributing/release_notes/)".

This PR won't be merged till the end of March. We'll update it every week to add new changelog files.
When we proofread all files from `changelogs/unreleased`, the PR will be ready for review.

### CODE REVIEW

File: changelogs/unreleased/luarocks-disable-force-config.md
Comment (Totktonada) [inline changelogs/unreleased/luarocks-disable-force-config.md:3]: While we're at it, it worth to drop the leading whitespace before an asterisk (at least for unification purposes).

Comment (arctic-dreamer) [reply changelogs/unreleased/luarocks-disable-force-config.md:3]: Sure, thanks for noticing!

File: changelogs/unreleased/add-check-for-user-input-of-the-number-of-iproto-threads.md
Comment (Totktonada) [inline changelogs/unreleased/add-check-for-user-input-of-the-number-of-iproto-threads.md:4]: There is a thing around hyphen/dash, which doubts me.

There are technical texts, which aim direct reading (without any pre-rendering) as a primary or a second goal (say, markdown supports rendering, but easy to read directly). The most obvious example is a comment in code. All or most of text that are tightly connected to code are also of this kind: they are usually written as plain text, asciidoc, markdown. Here non-ASCII symbols are generally discouraged. Here we expect a hyphen (or two hyphens), where in book we would see a dash.

There are texts, which primarily aim some rendering tool, which will prettify it, inline hyperlinks and convert to a format of a reader tools (generate HTML, PDF and so on). Those are texts for modern websites for example. Here we use Unicode en/em dash symbols or a special syntax to represent them using ASCII.

I feel release notes as a text of the first kind. Is the neural network in my head wrong here? What do you think?

Comment (xuniq) [reply changelogs/unreleased/add-check-for-user-input-of-the-number-of-iproto-threads.md]: I agree. So I am going to apply the rule from the doc [contribution guide](https://www.tarantool.io/ru/doc/latest/contributing/docs/style/#dashes) and use "---" as a dash. The documentation builder renders this character sequence in the source into em dash.

File: changelogs/unreleased/drop-net-box-console.md
Comment (Totktonada) [inline changelogs/unreleased/drop-net-box-console.md:8]: It is the note for a release manager. Please, keep it.

**Comment by Totktonada** [comment]
The patchset should meet several requirements to land into the `master` branch:

* Follow [our commit message guidelines](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message). Take care to 50/72 rules (lines length) and commits atomicity[^1]. Don't forget to add an appropriate prefix for commit titles: `doc:` is okay here.
* Fix all errors reported by the `lint / checkpatch` CI job. Take care to `NO_DOC`, `NO_TEST`, `NO_CHANGELOG` directives (look at examples in git log).
* If you aim to fix changelogs, which lands into `master` before a particular commit, please, point this commit. So I'll know what exactly I should review.

[^1]: I would like to see formatting/wording/meaning changes as separate commits, but I don't insist if it requires a lot of effort.

File: changelogs/unreleased/drop-net-box-console.md
Comment (Totktonada) [reply changelogs/unreleased/drop-net-box-console.md:8]: I suggest to run `./tools/gen-release-notes` to look at the resulting release notes. In particular, it'll show such release manager notes separately to highlight them somewhere at top.

File: changelogs/unreleased/gh-5385-tiny-tuples-v3.md
Comment (Totktonada) [inline changelogs/unreleased/gh-5385-tiny-tuples-v3.md:4]: Nit: It is better to don't make text much shorter than 80 symbols to make the entire changelog look visually same.

File: changelogs/unreleased/gh-5799-add-info-to-fiber-object.md
Comment (Totktonada) [inline changelogs/unreleased/gh-5799-add-info-to-fiber-object.md:4]: require(fiber) -> require('fiber')

File: changelogs/unreleased/gh-5941-datetime-type-support.md
Comment (Totktonada) [inline changelogs/unreleased/gh-5941-datetime-type-support.md:10]: Those bullets are still indented with a whitespace.

File: changelogs/unreleased/gh-6043-fiber-wakeup-self.md
Comment (Totktonada) [inline changelogs/unreleased/gh-6043-fiber-wakeup-self.md:9]: Lines breaks are a bit strange here.

File: changelogs/unreleased/fix-undeleted-unix-socket-path-and-multiple-socket-closing.md
Comment (xuniq) [reply changelogs/unreleased/fix-undeleted-unix-socket-path-and-multiple-socket-closing.md:5]: I have split them into separate bullets

File: changelogs/unreleased/gh-5819-election-triggers.md
Comment (Totktonada) [inline changelogs/unreleased/gh-5819-election-triggers.md:3]: (Same here.)

File: changelogs/unreleased/gh-5882-log-levels.md
Comment (Totktonada) [inline changelogs/unreleased/gh-5882-log-levels.md]: (Same here.)

File: changelogs/unreleased/gh-5941-datetime-type-support.md
Comment (Totktonada) [inline changelogs/unreleased/gh-5941-datetime-type-support.md]: (Same here, AFAIU we should use past simple.)

File: changelogs/unreleased/gh-5941-datetime-type-support.md
Comment (Totktonada) [inline changelogs/unreleased/gh-5941-datetime-type-support.md]: **Added** the methods?

File: changelogs/unreleased/gh-5941-datetime-type-support.md
Comment (Totktonada) [inline changelogs/unreleased/gh-5941-datetime-type-support.md]: (Same, should be past simple in my understanding.)

File: changelogs/unreleased/gh-6131-fix-crash-update-conflict.md
Comment (Totktonada) [inline changelogs/unreleased/gh-6131-fix-crash-update-conflict.md]: Nit: long line.

File: changelogs/unreleased/gh-6266-fix-crash-in-lua-triggers.md
Comment (Totktonada) [inline changelogs/unreleased/gh-6266-fix-crash-in-lua-triggers.md:5]: No empty line between paragraphs.

File: changelogs/unreleased/gh-6396-transaction-public-id.md
Comment (Totktonada) [inline changelogs/unreleased/gh-6396-transaction-public-id.md]: (In my understanding it should be in past simple.)

File: changelogs/unreleased/add-libunwind-submodule.md
Comment (Totktonada) [inline changelogs/unreleased/add-libunwind-submodule.md:8]: (Here too, should be past simple IMHO.)

File: changelogs/unreleased/gh-6789-fix-memleak-in-vdbe.md
Comment (Totktonada) [inline changelogs/unreleased/gh-6789-fix-memleak-in-vdbe.md:3]: *a* memory leak?

File: changelogs/unreleased/box-runtime-info-tuple.md
Comment (Totktonada) [inline changelogs/unreleased/box-runtime-info-tuple.md]: (*Added*)

File: changelogs/unreleased/fiber-parent-backtrace.md
Comment (Totktonada) [inline changelogs/unreleased/fiber-parent-backtrace.md:5]: (The same comment regarding past simple is applicable here too.)

File: changelogs/unreleased/gh-5815-memprof-enrich-symtab.md
Comment (Totktonada) [inline changelogs/unreleased/gh-5815-memprof-enrich-symtab.md]: Too long line.

File: changelogs/unreleased/gh-6043-fiber-wakeup-self.md
Comment (Totktonada) [inline changelogs/unreleased/gh-6043-fiber-wakeup-self.md]: Unusual line breaks. I don't know what a text editor you're using, but I use `gqap` to format a paragraph in Vim.

File: changelogs/unreleased/gh-6060-enable-backtrace-feature-on-aarch64.md
Comment (Totktonada) [inline changelogs/unreleased/gh-6060-enable-backtrace-feature-on-aarch64.md:3]: (I vote for past simple.)

File: changelogs/unreleased/gh-6150-memtx-mvcc-memory-monitoring.md
Comment (Totktonada) [inline changelogs/unreleased/gh-6150-memtx-mvcc-memory-monitoring.md:3]: (I vote for past simple.)

File: changelogs/unreleased/gh-6754-term-bump-before-promote.md
Comment (Totktonada) [inline changelogs/unreleased/gh-6754-term-bump-before-promote.md:3]: (I vote for past simple.)

File: changelogs/unreleased/gh-6930-isolation-level.md
Comment (Totktonada) [inline changelogs/unreleased/gh-6930-isolation-level.md:3]: (I vote for past simple.)

File: changelogs/unreleased/introduce-arm64-preliminary-support.md
Comment (Totktonada) [inline changelogs/unreleased/introduce-arm64-preliminary-support.md:5]: (I vote for past simple.)

File: changelogs/unreleased/update-libicu-static-build.md
Comment (Totktonada) [inline changelogs/unreleased/update-libicu-static-build.md:3]: (I vote for past simple.)

File: changelogs/unreleased/gh-6773-datetime-sql-builtin-functions.md
Comment (Totktonada) [inline changelogs/unreleased/gh-6773-datetime-sql-builtin-functions.md:3]: (I vote for past simple.)

File: changelogs/unreleased/elections-pre-vote.md
Comment (Totktonada) [inline changelogs/unreleased/elections-pre-vote.md]: (I vote for past simple.)

**Review by Totktonada — COMMENTED** [review]
The changes are generally good for me. I'm waiting for fixes and squashing changes.

I'll no re-verify it, so please resolve my comments once they're fixed.

**Review by Totktonada — APPROVED** [review]

**Pull request merged**
